backend/spec/spec_helper.rb

```diff
@@ -47,7 +47,7 @@
 Capybara.javascript_driver = :poltergeist
 
 # Set timeout to something high enough to allow CI to pass
-Capybara.default_wait_time = 10
+Capybara.default_wait_time = 35
```

Comment:
I don't think this should be so high.  Maybe 15 or 30 tops most servers like unicorn will timeout at 30 seconds, and once a request times out once there tends to be several so this will take much longer time to get a final build result.